cli/test/utils/tmpConfigPath.js

```diff
@@ -0,0 +1,14 @@
+const { tmpdir } = require('os');
+const { resolve } = require('path');
+
+module.exports = tmpConfigPath;
+
+/**
+ * Returns path where config could be temporary
+ * stored for test purposes
+ *
+ * @returns {string}
+ */
+function tmpConfigPath() {
```

Comment:
Could use [`tempy`](https://github.com/sindresorhus/tempy) to abstract away this helper.